config/fcos/v1_5_exp/translate_test.go

```diff
@@ -1632,3 +1632,28 @@ func TestTranslateGrub(t *testing.T) {
 		})
 	}
 }
+
+func TestIsPartitionValid(t *testing.T) {
+	disk := Disk{
+		Device:    "/dev/sdb",
+		WipeTable: util.BoolToPtr(false),
+		Partitions: []Partition{
+			{
+				Label:  util.StrToPtr("foo"),
+				Number: 0,
+			},
+			{
+				Label:  util.StrToPtr("bar"),
+				Number: 1,
+			},
+			{
+				Label:  util.StrToPtr("baz"),
+				Number: 2,
+			},
+		},
+	}
+
+	if err := isPartitionValid(disk); err != common.ErrInvalidPartition {
+		t.Errorf("Expected ErrInvalidPartition, but got %v", err)
+	}
+}
```

Comment:
When running this test for some reason it will only pass if a partition or partitions have the number 0 assigned to it..
will take a look and see where i went wrong